tidb-in-kubernetes/maintain-node.md

```diff
@@ -0,0 +1,233 @@
+---
```

Comment:
Please move this file to docs-cn/how-to/maintain/kubernetes-operations/ directory. Thanks!